src/current/v23.1/upgrade-cockroach-version.md

```diff
@@ -190,6 +190,8 @@ These steps perform an upgrade to the latest {{ page.version.version }} release,
     cp -i {COCKROACHDB_DIR}/cockroach /usr/local/bin/cockroach
     ~~~
 
+1. If a cluster has corrupt descriptors, a major-version upgrade cannot be finalized. In CockroachDB v23.1.6 and subsequent 23.1 versions, automatic descriptor repair is available but is disabled by default. In CockroachDB v23.1.11 and above, to enable automatic descriptor repair, set the environment variable `COCKROACH_RUN_FIRST_UPGRADE_PRECONDITION` to `true` after installing the v23.1 binary but before restarting the `cockroach` process on the node. Monitor the cluster logs for errors. If a descriptor cannot be repaired automatically, [contact support](https://support.cockroachlabs.com/hc) for assistance completing the upgrade.
```

Comment:
Maybe we should clarify that, "If you are a current CockroachDB customer and if a descriptor cannot be repaired...."

Reply:
I don't think we usually add this kind of language when inviting the reader to contact Support. I think customers whether or not they are customers. WDYT?

Reply:
That's fair.